packages/actionmenu/package.json

```diff
@@ -6,19 +6,22 @@
   "repository": "pluralsight/design-system",
   "author": "jaketrent",
   "keywords": [],
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
+  "main": "dist/esm/index.mjs",
```

Comment:
since we only have esm now; we could simplify this and just have paths like `dist/index.mjs`. just a thought - not necessary

Reply:
I think I'd rather not, preferring this for a couple reasons:

- We have other techs that aren't esm, such as the `css/` subdir that would be nice to be separate from.
- Leaves us space to re-intro `cjs/` if desired later.
- I think for anyone looking in `dist/`, it helps show what module types we do and do not publish.